src/note-components/Label/label.css

```diff
@@ -0,0 +1,59 @@
+.chips-container {
+    height: 13.75rem;
+    border: 1px solid rgba(0, 0, 0, .2);
```

Comment:
use color variable here

Reply:
This color is getting used only 1 time. That's why I'm not using variable for it.